src/pages/404.js

```diff
@@ -1,14 +1,9 @@
 import React from "react"
 
-import Layout from "../components/layout"
-import SEO from "../components/seo"
-
 const NotFoundPage = () => (
-  <Layout>
-    <SEO title="404: Not found" />
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-  </Layout>
+  <div class="notfound">
+    <a> NOT FOUND!</a>
```

Comment:
link with anchor? If no link then use text element instead

Reply:
Let's also push this to when we work on 404 page